ontology/fbcv.md

```diff
@@ -35,6 +35,7 @@ usages:
         description: "alleles and constructs annotated to bang sensitive in FlyBase"
 activity_status: active
 repository: https://github.com/FlyBase/flybase-controlled-vocabulary
+publications: []
```

Comment:
What is the point of documenting this? Just to say that there is no primary publication? Why is this useful?

Reply:
Having an empty list versus a missing value allows us to check if it’s been annotated explicitly that there’s no publication (yet)